builtin/providers/aws/resource_aws_instance.go

```diff
@@ -24,6 +24,9 @@ func resourceAwsInstance() *schema.Resource {
 		Update: resourceAwsInstanceUpdate,
 		Delete: resourceAwsInstanceDelete,
 
+		SchemaVersion: 1,
```

Comment:
Should we just jump straight to `2` here to make it easier to human-understand, or are we just going to go with zreo-indexed everything?